plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/callables/InlineNamedFunction.kt

```diff
@@ -26,4 +26,6 @@ class InlineNamedFunction(
         val paramNames = symbol.valueParameterSymbols.map { it.name }
         return ctx.insertInlineFunctionCall(signature, paramNames, args, body)
     }
+
+    override fun toViperMethod(): Nothing? = null
```

Comment:
Why `Nothing?` as return type and not `Method?`?

Reply:
`Nothing?` is more specific; we guarantee that inline methods do not generate a method.

Reply:
We can, `Nothing?` is a subtype of `Method?`, hence every value of type `Nothing?` (i.e. `null`) also has type `Method?`.  But `Nothing?` is more specific; e.g. someone doing `val m = toViperMethod()` and then trying `m as? Method` will receive a warning that this cast will never succeed, while if we return `Method?` we do not have a corresponding guarantee.

Reply:
Ok, thank you for the clarification!